tool/run/boot_dir/linux

```diff
@@ -31,32 +31,35 @@ proc run_boot_dir {binaries} {
 	# Save config part of the image.elf for easy inspection
 	exec cp -f [run_dir]/genode/config [run_dir].config
 
-        build_initrd $binaries
+        if {[board] == "pc"} {
 
-        file copy -force [genode_dir]/tool/boot/vmlinuz [run_dir]/vmlinuz
+            build_initrd $binaries
 
-        if {[have_include "image/iso"]} {
-            install_iso_bootloader_to_run_dir
-        }
+            file copy -force [genode_dir]/tool/boot/vmlinuz [run_dir]/vmlinuz
 
-        if {[have_include "image/uefi"]} {
+            if {[have_include "image/iso"]} {
+                install_iso_bootloader_to_run_dir
+            }
 
-            exec mkdir -p [run_dir]/efi/boot
-            exec cp [get_grub2_dir]/boot/grub2/grub2_32.efi [run_dir]/efi/boot/bootia32.efi
-            exec cp [get_grub2_dir]/boot/grub2/grub2_64.efi [run_dir]/efi/boot/bootx64.efi
-            exec mkdir -p [run_dir]/boot/grub
-        }
+            if {[have_include "image/uefi"]} {
 
-        set fh [open "[run_dir]/boot/grub/grub.cfg" "WRONLY CREAT TRUNC"]
-        puts $fh "set timeout=0"
-        puts $fh "menuentry 'Genode on Linux' {"
-        puts $fh " insmod linux"
-        puts $fh " linux /vmlinuz console=ttyS0,115200 amd_iommu=off intel_iommu=off"
-        puts $fh " initrd /initrd"
-        puts $fh "}"
-        close $fh
+                exec mkdir -p [run_dir]/efi/boot
+                exec cp [get_grub2_dir]/boot/grub2/grub2_32.efi [run_dir]/efi/boot/bootia32.efi
+                exec cp [get_grub2_dir]/boot/grub2/grub2_64.efi [run_dir]/efi/boot/bootx64.efi
+                exec mkdir -p [run_dir]/boot/grub
+            }
 
-        run_image
+            set fh [open "[run_dir]/boot/grub/grub.cfg" "WRONLY CREAT TRUNC"]
+            puts $fh "set timeout=0"
+            puts $fh "menuentry 'Genode on Linux' {"
+            puts $fh " insmod linux"
+            puts $fh " linux /vmlinuz console=ttyS0,115200 amd_iommu=off intel_iommu=off"
+            puts $fh " initrd /initrd"
+            puts $fh "}"
+            close $fh
+
+            run_image
+        }
```

Comment:
Oh, this addresses my first remark. Please merge this into a1a765d298d686402e0114084be95d9416d13f1e